app/javascript/controllers/chart_controller.js

```diff
@@ -3,7 +3,8 @@ import { Controller } from "@hotwired/stimulus"
 import { Chart, registerables } from "chart.js";
 Chart.register(...registerables);
 
-// import "chartjs-plugin-colorschemes";
+import ChartDataLabels from "chartjs-plugin-datalabels";
```

Comment:
It's better to have all imports at the top after each other

Reply:
Do we need this in this chart at all?

Reply:
I think there's a bug in the plugin, even if I didn't incorporate the label functionality in this chart, it showed the value for every point also in here. To disable the labels, I had to import it here too and disable labels later in the code

Reply:
Hmm. Maybe if you remove the imports from `app/javascript/application.js`? Or does that lead to an error?

Reply:
problem still there... 
<img width="991" alt="Screenshot 2022-01-27 at 11 42 47" src="https://user-images.githubusercontent.com/32681439/151343411-177487b2-6eda-4003-b39f-273c1ad51892.png">

Reply:
now I register the plugin in the showChart function and the problem in the events chart is gone!